File: WEB + CodenameOne/src/Entity/ReservationVol.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * ReservationVol
 *
 * @ORM\Table(name="reservation_vol")
 * @ORM\Entity
 */
class ReservationVol
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_resV", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idResv;

    /**
     * @var int
     *
     * @ORM\Column(name="id_user", type="integer", nullable=false)
     */
    private $idUser;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut", type="date", nullable=false)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_fin", type="date", nullable=false)
     */
    private $dateFin;

    /**
     * @var float
     *
     * @ORM\Column(name="prix_total", type="float", precision=10, scale=0, nullable=false)
     */
    private $prixTotal;

    /**
     * @var int
     *
     * @ORM\Column(name="id_vol", type="integer", nullable=false)
     */
    private $idVol;

    /**
     * @var int
     *
     * @ORM\Column(name="nbrpsg", type="integer", nullable=false)
     */
    private $nbrpsg;

    /**
     * @var string
     *
     * @ORM\Column(name="aerocomp", type="string", length=20, nullable=false)
     */
    private $aerocomp;

    /**
     * @var string
     *
     * @ORM\Column(name="classe", type="string", length=20, nullable=false)
     */
    private $classe;

    public function getIdResv(): ?int
    {
        return $this->idResv;
    }

    public function getIdUser(): ?int
    {
        return $this->idUser;
    }

    public function setIdUser(int $idUser): self
    {
        $this->idUser = $idUser;

        return $this;
    }

    public function getDateDebut(): ?\DateTimeInterface
    {
        return $this->dateDebut;
    }

    public function setDateDebut(\DateTimeInterface $dateDebut): self
    {
        $this->dateDebut = $dateDebut;

        return $this;
    }

    public function getDateFin(): ?\DateTimeInterface
    {
        return $this->dateFin;
    }

    public function setDateFin(\DateTimeInterface $dateFin): self
    {
        $this->dateFin = $dateFin;

        return $this;
    }

    public function getPrixTotal(): ?float
    {
        return $this->prixTotal;
    }

    public function setPrixTotal(float $prixTotal): self
    {
        $this->prixTotal = $prixTotal;

        return $this;
    }

    public function getIdVol(): ?int
    {
        return $this->idVol;
    }

    public function setIdVol(int $idVol): self
    {
        $this->idVol = $idVol;

        return $this;
    }

    public function getNbrpsg(): ?int
    {
        return $this->nbrpsg;
    }

    public function setNbrpsg(int $nbrpsg): self
    {
        $this->nbrpsg = $nbrpsg;

        return $this;
    }

    public function getAerocomp(): ?string
    {
        return $this->aerocomp;
    }

    public function setAerocomp(string $aerocomp): self
    {
        $this->aerocomp = $aerocomp;

        return $this;
    }

    public function getClasse(): ?string
    {
        return $this->classe;
    }

    public function setClasse(string $classe): self
    {
        $this->classe = $classe;

        return $this;
    }


}

File: DESKTOP/src/entities/ReservationVol.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;
import entities.Vol;
import entities.Reservation;
import java.sql.Date;
import java.util.logging.Logger;

public class ReservationVol extends Reservation{
   
  private int id_resV,id_vol,nbrpsg;
  private String classe,aerocomp;
private double prix_v;  
public ReservationVol(int id_resV, int id_user, Date date_debut, Date date_fin, double prix_total,int id_vol,int nbrpsg ,String aerocomp ,String classe ){
      
     this.id_resV = id_resV;
        this.id_vol = id_vol;
       // this.prix_v = prix_v;
        this.prix_total = prix_total;
        this.nbrpsg = nbrpsg;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.id_user = id_user;
        this.classe = classe;
        this.aerocomp = aerocomp;
                        

}

   public ReservationVol(int id_user, Date date_debut, Date date_fin, double prix_total,int id_vol,int nbrpsg ,String aerocomp ,String classe ){
      
    
        this.id_vol = id_vol;
        this.prix_total = prix_total;
        this.nbrpsg = nbrpsg;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.id_user = id_user;
        this.classe = classe;
        this.aerocomp = aerocomp;
                        

}

    public ReservationVol() {
        
    }



   

    public int getId_resV() {
        return id_resV;
    }

    public void setId_resV(int id_resV) {
        this.id_resV = id_resV;
    }

    public int getId_vol() {
        return id_vol;
    }

    public void setId_vol(int id_vol) {
        this.id_vol = id_vol;
    }

    public int getNbrpsg() {
        return nbrpsg;
    }

    public void setNbrpsg(int nbrpsg) {
        this.nbrpsg = nbrpsg;
    }

    public String getClasse() {
        return classe;
    }

    public void setClasse(String classe) {
        this.classe = classe;
    }

    public String getAerocomp() {
        return aerocomp;
    }

    public void setAerocomp(String aerocomp) {
        this.aerocomp = aerocomp;
    }

    public double getPrix_v() {
        return prix_v;
    }

    public void setPrix_v(double prix_v) {
        this.prix_v = prix_v;
    }



    
}
